tests/conftest.py

```diff
@@ -164,23 +166,24 @@ def teardown():
 
 
 @pytest.fixture(scope="session")
-def registry(minikube, worker_id):  # pylint: disable=redefined-outer-name
+def registry(minikube, worker_id, request):  # pylint: disable=redefined-outer-name
     def get_registry_logs():
         cont.reload()
         return cont.logs().decode("utf-8")
 
     cont = None
     port = None
-    if worker_id in ("master", "gw0"):
+    k8s_container = request.config.getoption("--k8s-container")
+    if not k8s_container and worker_id in ("master", "gw0"):
         minikube.start_registry()
         port = minikube.registry_port
     print("\nWaiting for registry to be ready ...")
     assert wait_for(
-        p(container_is_running, minikube.client, "registry"), timeout_seconds=60
+        p(container_is_running, minikube.client, "registry"), 60, 2
```

Comment:
Can you use keyword args here and everywhere that passes both the timeout and interval?  It is much more readable that way even if it is a bit verbose.